### PR TITLE
feat(balance): Increase earlygame zombie damage rolls

### PR DESCRIPTION
**Pull request by RoyalFox2140**
### Required

- [x] I wrote the PR title in [conventional commit format](https://docs.cataclysmbn.org/en/contribute/changelog_guidelines/).
- [x] I ran the [code formatter](https://docs.cataclysmbn.org/en/contribute/contributing/#code-style).
- [x] I linked any relevant issues using [github keyword syntax](https://docs.cataclysmbn.org/en/contribute/contributing/#pull-request-notes) so it can be closed automatically.
- [x] I have [committed my changes to new branch that isn't `main`](https://docs.cataclysmbn.org/en/contribute/contributing/#make-your-changes) so it won't cause conflict when updating `main` branch later.

## Purpose of change

Zombies do very little damage in BN, combined with dodge being easy to train and very effective along with clothing having fairly high stats (3 armor on jeans, 6 on fur and leather coats, plus crafted armor such as gambesons or survivor gear), and high stun chance with a blunt weapon that has good hit chance and low swing speed, it's fairly easy to avoid meaningful damage.

## Describe the solution

Add another hit die to a large number of tier 0 zombies, and some evolutions. This will put normal zombies from max damage 6 to 9, but this has other remarkable effects across the whole game: 

Each increase in bash damage will mean zombies have an easier time breaking terrain to get to you
This is more damage to armor and weapons which requires repair work
More required use of first aid skill books especially earlygame when players normally can completely ignore it if you're not picking an injured start.

## Describe alternatives you've considered

Additional areas to tweak include dodge chance, enemy hit chance, healing rate, armor protection values, grab strength, and stamina loss penalties.

## Testing

I've done massively harsher penalties over at the Deadly Zombies addon and so far it feels players can handle the tweaks. The tweaks will also let us re-introduce some of the lost firearm spawns without worry of further trivialization.

## Additional context

More to follow, as we see how each PR works out.

### CODE REVIEW

**Review by chaosvolt — APPROVED** [review]
Looks good from what I can tell, but leaving this open for others to look at.

**Comment by Soadreqm** [comment]
>and high stun chance with a blunt weapon that has good hit chance and low swing speed

I feel like early game zombies should _not_ be balanced against the cudgel. Yes, you can make it as soon as you get a knife (which many characters start the game with), but at the same time, it's the absolute pinnacle of early game melee weapons, the automatic top pick for anyone who understands the meta and isn't trying to roleplay.

In the hands of a skilled wielder, it can 1v1 any non-ranged enemy up to a kevlar zombie. And testing it at 5 melee and bashing, the cudgel can still _damage_ a kevlar zombie on a crit, it's just that an unarmored survivor with 5 dodge cannot survive the fight long enough to kill it. Soldiers and brutes have no change, they get stunlocked and melt away. In the hands of an unskilled wielder it's not quite as terrifying, but it's better at dealing with early game threats than any other weapon you can make. Well, except maybe the knife spear if you use it from a rooftop, but crafting that is a lot more work.

In short the cudgel is crazy OP, and I'm afraid that if zombies are made stronger to compensate, that's going to make most non-cudgel early game weapons no longer viable.

>Each increase in bash damage will mean zombies have an easier time breaking terrain to get to you

I don't think you want to overdo this either. Blocking a zombie's path with by closing doors or dragging furniture _should_ work to slow it down. Also, you gotta mind the threshold for being able to Kool-Aid Man through walls. Survivor zombies can do it, and that already seems a bit over the top.

**Comment by RoyalFox2140** [comment]
1: All crits stun enemies. 
2: Dodge chances is very high without a weapon
3: Most weapons can take on 1-2 zombies at a time, punishing players for tanking 4 zombies at once is what I feel intended game behavior should be.
4: Armor is still easily layered
5: Melee bonuses scale dramatically, melee is fairly easy to train with minimal danger by attacking passive wildlife or singular targets, including zombies but also random dogs.

6: And most importantly, a revert is a single button away so anyone can revert if this is undesired behavior or too challenging. Doing nothing is a worse solution because there are way too many difficulty problems currently, BN has lost control of it's difficulty curve. I prefer we experimented with potential solutions to reach the best outcome.

All this is layed out on the Discord thread, for greater context:
https://discord.com/channels/830879262763909202/1300866082160115752/1300866084332896367
